app/templates/public/exhibition/view.hbs

```diff
@@ -22,10 +24,10 @@
             </div>
         {{/if}}
         <br/><br/>
-        <h1 class="ui centered header no margin">
-            <img class="ui small image m-0 mb-1" src="{{or this.model.exhibitor.logoUrl '/images/placeholders/Other.jpg'}}" alt="Logo">
-            {{this.model.exhibitor.name}}
-        </h1>
+        <div class="d-flex items-center content-center">
```

Comment:
no need of this

Reply:
Mam, Actually If I don't go this way and simply change the dimensions of image, then Image and text does not seem to be  aligned vertically. I thought that might be a problem. If its not I can revert these changes. Please tell me should I revert them.

Reply:
then add padding b/w them

Reply:
Sure Mam.